src/rpc/ice.js

```diff
@@ -1496,11 +1496,13 @@ function listen_on_port_range(port_range) {
         server.listen(port);
         // wait for listen even, while also watching for error/close.
         return promise_utils.wait_for_event(server, 'listening')
-            .return(server)
+            .then(function() {
+                return server;
+            })
```

Comment:
you don't want to define functions in promise chains because any access to `this` will not work, so prefer arrow functions in chains:
```suggestion
            .then(() => server)
```